client/python/_fastrand.c

```diff
@@ -36,10 +36,12 @@ limitations under the License.
 
 uint64_t randbits(float p1, int num_bits) {
   uint64_t result = 0;
-  int threshold = (int)(p1 * RAND_MAX);
+  uint64_t threshold = p1 * RAND_MAX;
   int i;
   for (i = 0; i < num_bits; ++i) {
-    uint64_t bit = (rand() < threshold);
+    // NOTE: The comparison is <= so that p1 = 1.0 implies that the bit is
```

Comment:
This comment does not reflect what the change to the code is doing, and both are wrong.
p=1.0 implies that the value is always zero. I think you meant to type p=0 implies the bit is always one. That would be true given the change you made. However it is undesirable. We don't want it to be the case that if someone sets p=0 they get a bit array of all 1s. It should be zeros. Less than is correct.